src/snowflake/snowpark/mock/_functions.py

```diff
@@ -887,6 +887,13 @@ def mock_to_array(expr: ColumnEmulator):
     return res
 
 
+@patch("strip_null_value")
+def mock_strip_null_value(expr: ColumnEmulator):
+    return ColumnEmulator(
+        [None if x == "null" else x for x in expr], sf_type=expr.sf_type
+    )
```

Comment:
I feel like nullability could be different than the input column?

Reply:
same question about the nullability of the ColumnType, is the current backend behavior always returning true?

Reply:
Experimentally it appears to always return a column that is nullable. I wonder if it would be useful to create a test that runs both the normal version and local testing version of a test and then ensures they return the same result.

Reply:
Tests outside `tests/mock` that are marked with `pytest.mark.localtest` are run against both a live connection and a local testing connection.